exercises/practice/perfect-numbers/.approaches/switch-expression/content.md

```diff
@@ -0,0 +1,62 @@
+# `switch` expression
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public enum Classification
+{
+    Perfect,
+    Abundant,
+    Deficient
+}
+
+public static class PerfectNumbers
+{
+    public static Classification Classify(int number)
+    {
+        var sum = Enumerable.Range(1, number - 1)
+            .Sum(n => number % n == 0 ? n : 0);
+            
+        return (sum < number, sum > number) switch
+        {
+            (true, _) => Classification.Deficient,
+            (_, true) => Classification.Abundant,
+            _ => Classification.Perfect,
+        };
+    }
+}
+```
+
+This approach (in short):
+1. Takes a _range_ of numbers starting from the number `1` up until (but not including) the number being classified.
+2. Keeping only the numbers which are a _factor_ of the number being classified.
+3. Summing these factors and classifying this sum according to given instructions.
+
+Let's go through this in more detail:
+
```

Comment:
I recommend writing one sentence per line. This might be policy as well.

Reply:
Ah, great point. I'll take this into account.